### ISSUE TITLE
Pylint doesn't recognise methods defined on enum.Enum subclasses

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#301]
Originally reported by: **Phil Connell (BitBucket: [pconnell](http://bitbucket.org/pconnell), GitHub: @pconnell?)**

---

Running pylint 1.3 over the code below produces a couple of spurious warnings. The no-member is the main problem.

enum is going to be increasingly widely used: while the stdlib version is only in 3.4, it's backported to 2.\* and 3.*. Projects using it are faced with having to disable "no-member", which devalues pylint significantly.

So would be nice to see either of:
- Pylint growing more intelligence w.r.t. enums so that methods are recognised.
- A new config item that allows no-member checks to be skipped on all _subclasses_ of Enum.

```
#!shell

$ pylint -r n test.py
No config file found, using default configuration
************* Module test
W:  4, 0: Class has no __init__ method (no-init)
E: 12, 8: Instance of 'abc' has no 'meth' member (no-member)

$ pylint --version
No config file found, using default configuration
pylint 1.3.0, 
astroid 1.2.0, common 0.62.1
Python 3.4.1 (default, May 19 2014, 17:23:49) 
[GCC 4.9.0 20140507 (prerelease)]
```

```
#!python

"""..."""

import enum
class XYZ(enum.Enum):
    """..."""
    abc = 1

    def meth(self):
        """..."""
        pass

XYZ.abc.meth()

```

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/301
